namer/consul/src/test/scala/io/buoyant/namer/consul/SvcAddrTest.scala

```diff
@@ -113,4 +96,21 @@ class SvcAddrTest extends FunSuite with Matchers {
     Await.ready(retried, 1.second)
     numOfRequests.intValue() should equal(numOfAttempts)
   }
+
+  test("should be Addr.Neg unexpected error occurs") {
+    // givenn
+    val invoked = Promise[Unit]()
+    val api = apiStub { (_, _, _, _, _, _) =>
+      invoked.setDone()
+      Future.exception(new Throwable("No path to datacenter"))
+    }
+
+    // when
+    val addr: Var[Addr] = SvcAddr(api, hangForLongBackoff, "dc1", SvcKey("svc", None), None, None, None, Map.empty, Stats(NullStatsReceiver))
+    addr.changes.respond(_ => ())
+
+    // then
+    Await.ready(invoked, 1.second)
```

Comment:
I think this should be `await`

Reply:
oh, I just noticed all the Await.readys in this file.  I think... they should all be `await` instead?